pkg/apis/componentconfig/types.go

```diff
@@ -230,7 +230,7 @@ type KubeletConfiguration struct {
 	// be rejected.
 	LowDiskSpaceThresholdMB int `json:"lowDiskSpaceThresholdMB"`
 	// How frequently to calculate and cache volume disk usage for all pods
-	VolumeStatsAggPeriod unversioned.Duration `json:volumeStatsAggPeriod`
+	VolumeStatsAggPeriod unversioned.Duration `json:"volumeStatsAggPeriod"`
```

Comment:
@mikedanese Is anyone using these types yet?

Clearly we need a checker for this.